Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml

```
@@ -37,6 +37,12 @@ configuration:
   - Fetch Correlation Events Only
   required: true
   type: 15
+- defaultvalue: '20'
+  display: Maximum number of events to pull per incident.
+  name: events_limit
+  required: false
+  type: 0
+  additionalinfo: Relevant for ‘long running instance’.
```

```suggestion
- defaultvalue: '20'
  display: Maximum number of events per incident.
  name: events_limit
  required: false
  type: 0
  additionalinfo: The maximal amount of events to pull per incident.
```
In v2 there was an option to run with long-running or not, in your case it's only long-running